source/MapOutfitterPanel.h

```diff
@@ -54,7 +54,12 @@ class MapOutfitterPanel : public MapSalesPanel {
 	
 	
 private:
+	// Maps category names to list of outfits in that category, sorted
+	// alphabetically by name.
+	// 
+	// TODO: s/vector/set/?  The vector is anyway sorted and deduplicated.
```

Comment:
Adding this TODO feels a little bit strange. This replace sounds like a good idea (based on your explanation above), but it also looks like the type of comment one would make in a 1-person project for something to investigate in the future. ES is not a 1-person project and others reading the code will not look at this TODO the same way as you would.

I also expect that not every C++ programmer in ES will be familiar with the replace command syntax as used in VI and sed. So when adding comments I suggest to use regular English instead of replace command syntax.

Reply:
Rewritten in 7b7459b3054894466f79ee91752efacf359c5074.

Reply:
Good, please press the "Mark as resolved" button for this conversation. (I don't have permissions to do that.)

Reply:
Done, and thanks for all the review feedback so far, @petervdmeer! :+1: